client/init.c

```diff
@@ -103,6 +103,15 @@ TDNFCloneCmdArgs(
                                    &pCmdArgs->pSetOpt);
         BAIL_ON_TDNF_ERROR(dwError);
     }
+    else /* if there are no setopt values, prime it to ensure non null */
+    {
+        dwError = AddSetOptWithValues(
```

Comment:
I could not understand the use of this dummy option and values. Could you please help me understand what would be the use of this?

Reply:
A clone will not happen if there were no setopt from command line. In this case, further operations with the internal setopt clone would fail. Hence priming it with dummy values